src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -251,6 +230,7 @@ static Task StopEndpoints(IEnumerable<ComponentRunner> endpoints)
             var stopTimeout = TimeSpan.FromMinutes(2);
             return endpoints.Select(async endpoint =>
             {
+                await Task.Yield(); // ensure all endpoints are stopped even if a synchronous implementation throws
```

Comment:
I get your point but I find it a bit brute force and since we are stopping I'd say YAGNI since it might also delay the stop procedure